src/tests/test_dynamic_api/test_filters.py

```diff
@@ -436,3 +436,74 @@ def test_filter_isempty(parkeervakken_parkeervak_model, filled_router):
         data = read_response_json(response)
         assert len(data["_embedded"]["parkeervakken"]) == 1
         assert data["_embedded"]["parkeervakken"][0]["id"] == "121138489006"
+
+    @staticmethod
```

Comment:
I do realize that you are merely following the existing code @DonQueso89 , but why are these tests in a class and is each test function annotated with `@staticmethod`? Pinging @vdboor  

 We are using `pytest` here. Hence we can do away with the `unittest`-isms.

I say ditch the class and the `@staticmethod` annotations.

Reply:
I leave this for discussion